packages/alea-frontend/components/nap/GradingInterface.tsx

```diff
@@ -515,13 +517,17 @@ export function GradingInterface({ courseId }: { courseId: string }) {
     sortOrders: DEFAULT_SORT_ORDER,
   });
   const [gradingItems, setGradingItems] = useState<GradingItem[]>([]);
+  const originalGradingItems = useRef<GradingItem[]>([]);
   const homeworkMap = useRef<Record<string, HomeworkInfo>>({});
   const questionMap = useRef<Record<string, Problem>>({});
 
   const [selected, setSelected] = useState<
     { homeworkId: number; questionId: string; studentId: string } | undefined
   >(undefined);
 
+  const [showHomework, setShowHomework] = useState(true);
```

Comment:
Add these to `sortAndFilterParams` instead of creating new state. and name them `showHomeworkOnly` and `showPracticeOnly`